others_recipes/afangsoup.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Afang Soup</h1>
+    <img src="whitesoup_images/afang soup.jpg" />
+    <h2>Description</h2>
+    <p>
+      Afang soup is a tasty, nutritious Nigerian soup, native to the
+      Efiks/Ibibios (Akwa Ibom & Cross river states).
+    </p>
+    <h2>Ingredients</h2>
+    <ul>
+      <li>Beef</li>
+      <li>Fish</li>
+      <li>Palm oil</li>
+      <li>Crayfish</li>
+      <li>Pepper</li>
+      <li>Shaki (cow tripe)</li>
+      <li>Waterleaf</li>
+      <li>Okazi leaf</li>
+      <li>Onion</li>
+      <li>Periwinkle</li>
+      <li>Salt</li>
+      <li>Some seasoning</li>
+    </ul>
+    <h1>Steps</h1>
```

Comment:
- [ ] It is best practice to have only one `h1` tag on a page for SEO (search engine optimization) reasons. You can use `h2` or `h3` instead. Please [read more about this here.](https://www.investisdigital.com/blog/seo/using-h1-tag-improves-search-engine-ranking)
However, there are some ongoing argument on this topic but it is good you stick to the convention until it become clear . Read [about google stand on this](https://www.searchenginejournal.com/h1-headings-for-google/406720/#close).